src/components/TodoList.tsx

```
@@ -11,17 +15,41 @@ type TodoListProps = {
-           :
-            <TodoItem key={todo.id} todo={todo} onDeleteTodo={onDeleteTodo} onCompleteToggle={onCompleteToggle} onEditTodoToggle={onEditTodoToggle} />
-          }
-        </>
-      ))}
-    </ul>
+    <DndContext sensors={sensors} collisionDetection={closestCenter} onDragEnd={handleTodoDragEnd}>
+      <SortableContext items={todos} strategy={verticalListSortingStrategy}>
+        <ul className="mt-4">
+          {todos.map((todo: Todo) => 
+            todo.isEdit ? 
+              <EditTodoNameForm id={todo.id} name={todo.name} onUpdateTodoName={onUpdateTodoName} />
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)